### ISSUE TITLE
Gatsby / Gatsby Source Craft Issues: Fields "min" conflict because they return conflicting types Int and Float.

### ISSUE THREAD

**Issue by zjr** [verbb/formie#1188]
### Describe the bug

I'm not seeing anything interesting in my Craft logs but maybe I'm just missing it. Sorry I can't be more helpful.

```
"gatsby-source-craft" threw an error while running the sourceNodes lifecycle:

Failed to execute query LIST_testForm_Form.
Errors:
Fields "min" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "min" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Int and Float. Use
different aliases on the fields to fetch both if this was intentional.

Fields "max" conflict because they return conflicting types Float and Int. Use
different aliases on the fields to fetch both if this was intentional.

  517 |     if (remoteConfigVersion !== localConfigVersion || !localContentUpdateTime)
 {
  518 |         reporter.info("Cached content is unavailable or outdated, sourcing 
_all_ nodes.");
> 519 |         await sourceAllNodes(config);
      |         ^
  520 |     }
  521 |     else {
  522 |         reporter.info(`Craft config version has not changed since last 
sourcing. Checking for content changes since "${localContentUpdateTime}".`);

File: node_modules/gatsby-source-craft/gatsby-node.js:519:9



  Error: Failed to execute query LIST_testForm_Form.
  Errors:
  Fields "min" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "min" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Int and Float. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  Fields "max" conflict because they return conflicting types Float and Int. Use diffe
  rent aliases on the fields to fetch both if this was intentional.
  
  - paginate.ts:60 Object.paginate
    [heparks-client]/[gatsby-graphql-source-toolkit]/src/source-nodes/fetch-nodes/pagi
    nate.ts:60:13
  
  - task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5
  
  - fetch-lists.ts:61 fetchNodeList
    [heparks-client]/[gatsby-graphql-source-toolkit]/src/source-nodes/fetch-nodes/fetc
    h-lists.ts:61:20
  
  - fetch-lists.ts:36 Object.fetchAllNodes
    [heparks-client]/[gatsby-graphql-source-toolkit]/src/source-nodes/fetch-nodes/fetc
    h-lists.ts:36:24
  
  - create-nodes.ts:13 Object.createNodes
    [heparks-client]/[gatsby-graphql-source-toolkit]/src/source-nodes/node-actions/cre
    ate-nodes.ts:13:20
  
  - async Promise.all
  
  - source-all-nodes.ts:20 sourceAllNodes
    [heparks-client]/[gatsby-graphql-source-toolkit]/src/source-nodes/source-all-nodes
    .ts:20:3
  
  - gatsby-node.js:519 Object.exports.sourceNodes
    [heparks-client]/[gatsby-source-craft]/gatsby-node.js:519:9
  
  - api-runner-node.js:487 runAPI
    [heparks-client]/[gatsby]/src/utils/api-runner-node.js:487:16
```

### Steps to reproduce

1. Try to run `gatsby develop` in a project with Formie and at least one form with input (I get other errors when there aren't any forms/inputs). Relevant software versions listed below…

### Form settings

- Multi-page form: No
- Submission Method: N/A
- Client-side Validation: N/A
- Custom Form Templates: No

### Craft CMS version

4.3.3

### Plugin version

2.0.18

### Multi-site?

No

### Additional context

gatsby: 4.24.8
gatsby-source-craft: 3.0.0
node 16.18.1

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by zjr** [comment]
Seems to be working, thanks a lot!

**Comment by engram-design** [comment]
Fixed in [2.0.19](https://github.com/verbb/formie/releases/tag/2.0.19)
